roxctl/central/debug/download_diagnostics.go

```diff
@@ -41,14 +44,20 @@ func downloadDiagnosticsCommand(cliEnvironment environment.Environment) *cobra.C
 			if urlParams != "" {
 				path = fmt.Sprintf("%s?%s", path, urlParams)
 			}
-			return zipdownload.GetZip(zipdownload.GetZipOptions{
+			err := zipdownload.GetZip(zipdownload.GetZipOptions{
 				Path:       path,
 				Method:     http.MethodGet,
 				Timeout:    flags.Timeout(c),
 				BundleType: "diagnostic",
 				ExpandZip:  false,
 				OutputDir:  outputDir,
 			}, cliEnvironment.Logger())
+			if isTimeoutError(err) {
+				cliEnvironment.Logger().ErrfLn(`
```

Comment:
Skip the first newline (i.e. moving the ` to the next line), otherwise the error would look a bit weird IMO:
```
ERROR:
Timeout is reached while creating...
```